drivers/hmis/app/graphql/mutations/delete_client_alert.rb

```diff
@@ -0,0 +1,25 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Mutations
+  class DeleteClientAlert < CleanBaseMutation
+    argument :id, ID, required: true
+
+    field :client_alert, Types::HmisSchema::ClientAlert, null: true
+    field :errors, [Types::HmisSchema::ValidationError], null: false, resolver: Resolvers::ValidationErrors
+
+    def resolve(id:)
+      record = Hmis::ClientAlert.find(id)
+      raise 'Access denied' unless allowed?(permissions: [:can_manage_client_alerts])
```

Comment:
we should specify the "entity" for the permissions here, which is the client

```suggestion
      raise 'Access denied' unless current_permission?(permission: :can_manage_client_alerts, entity: record.client)
```

Reply:
Thank you!